Quests-0.1.10

planetme opened the pull request:
Scheduled merge of quest fixes and changes before 0.1.10 release.

spacesailormarie committed:
tiny little quest fix (#561)

* Add files via upload

serializationisbad json that magically keeps failing to autodownload

* fixed a few typos and inaccuracies in quests

* add to manifest

* remove from moddirector, move json to config

* Update Quest

fixed inaccuracy

* fixed high purity iron quest description

* update microprocessor quest

* fixed nickel zinc ferrite quest

* fixed coke oven quest typo

---------

Co-authored-by: planetme <87911459+planetme@users.noreply.github.com>
Co-authored-by: bruberu <80226372+bruberu@users.noreply.github.com>

planetme committed:
Merge branch 'master-ceu' into master-quests

spacesailormarie committed:
fixed antimony quest (#570)

spacesailormarie committed:
another one (#578)

* fixed antimony quest

* fixed littletiles quest

* fixed industrial renewal quest

spacesailormarie committed:
fix lst and lbb quests (#583)

* fixed lst and lbb quests

* uhh umm uh readd lbb dependencies

* fix error because i am incomprehensibly stupid

spacesailormarie committed:
big fat quest fix (#586)

* fixed antimony quest

* fixed lst and lbb quests

* uhh umm uh readd lbb dependencies

* fix error because i am incomprehensibly stupid

* lst and electrolyzer fix

* lst

* water pump and sintering oven fix

* its caesium not cesium

* idk at this point bro

* THE JOKER

* chapters fixed + no more edit mode

* BED QUESTLINE

---------

Co-authored-by: planetme <87911459+planetme@users.noreply.github.com>

ZombieZilla committed:
Bed quest positioning (#589)

planetme committed:
Revert "Bed quest positioning (#589)"

This reverts commit 12900fc84d032ea82e0a798f592afe7216d9a6be.

planetme committed:
add beds for real

planetme committed:
Tungsten, Molybdenum, and Rhenium

planetme committed:
Merge branch 'master-ceu' into master-quests

TechLord22 commented on config/betterquesting/DefaultQuests.json:
```diff
@@ -57244,7 +57254,7 @@
                 }
               },
-              "OreDict:8": "",
+              "OreDict:8": "Finally, with the Rayon, Memory foam, some wood, and steel springs, you can get a good night\u0027s sleep.\n\nYou §l§n§r§l§odo need EV power§r to run this recipe though.\n\n§o§6Have a good night.",
```
Is changing this for the oredict field intentional?

planetme committed:
fix

planetme committed:
Merge branch 'master-ceu' into master-quests

planetme committed:
fix recipe conflict (READ DESCRIPTION)

before you guys execute me for putting a hotfix into master-quests, this is sort of an urgent fix required for 0.1.10, and master-hotfixes was merged already...

planetme committed:
minor grammatical error

planetme committed:
Merge branch 'master-ceu' into master-quests

bruberu reviewed (approved):
seems fine

planetme committed:
AR removal compatability, removed blank quest
